08_files_lab/01.py

```diff
@@ -1,6 +1,17 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one
-"""
-
+"""
+      _      _      _
+   __(.)< __(.)> __(.)=
+   \___)  \___)  \___)   
+
+Python course - Verint
+Week 2
+
+"""
+
+import sys
+
+with open(sys.argv[1], "r") as fin:
+    with open(sys.argv[2], "a") as fout:
+        fout.write("\n")                

```

Comment:
Only if the file does not end with a '\n' (which is NOT commonly the case). Many editors will automatically add a '\n' to the end of the last line in the file.
If you insist on handling files that don't have a newline at their end it's better to check for the (non)existence of the newline to prevent double newline problem